_includes/cockroachcloud/free-throttle.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_info}}
+Free clusters have heavily throttled resources and limited features. If you want a cluster with higher performance and full capabilities, you can [create a paid Serverless cluster](create-a-serverless-cluster.html) for as little as $1 per month.
```

Comment:
They're really not heavily throttled, they're actually awesome!! 😅

Reply:
I might emphasize here that you can always create a free cluster and add budget later; you don't need to create a new cluster.

Reply:
Oops yeah I removed mentions of this include after the cluster unification switch but forgot to delete the original file.  ty!